internal/unionstore/memdb.go

```diff
@@ -162,6 +163,9 @@ func (db *MemDB) Reset() {
 	db.vlog.reset()
 	db.allocator.reset()
 }
+func (db *MemDB) SetDiskFullOpt(level kvrpcpb.DiskFullOpt) {
```

Comment:
Please add a empty line before a function, and leave a space after "//".

Reply:
Done!